web-console/src/views/query-view/run-button/run-button.tsx

```diff
@@ -151,25 +153,34 @@ export class RunButton extends React.PureComponent<RunButtonProps> {
 
   render(): JSX.Element {
     const { runeMode, onRun, loading } = this.props;
-    const runButtonText = 'Run' + (runeMode ? 'e' : '');
 
     return (
       <ButtonGroup className="run-button">
         {onRun ? (
           <Tooltip content="Control + Enter" hoverOpenDelay={900}>
             <Button
+              className={runeMode ? 'rune-button' : ''}
```

Comment:
Doing `className={runeMode ? 'rune-button' : undefined}` here would be more idiomatic.

Reply:
Done